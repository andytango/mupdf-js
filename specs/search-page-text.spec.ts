import { forExamplePages, loadExampleFile } from "./helpers/example-file";

describe("searchPageText", () => {
  forExamplePages("should search the page text", async (page) => {
    const { mupdf, doc } = await loadExampleFile();
    expect(mupdf.searchPageText(doc, page, "Lorem", 10)).toMatchSnapshot();
  });
});
