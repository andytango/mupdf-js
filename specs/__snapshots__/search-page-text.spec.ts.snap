// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchPageText Page [1 / 4] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText Page [2 / 4] - should search the page text 1`] = `
[
  {
    "h": 17.862,
    "isContinuation": true,
    "w": 36.84198,
    "x": 130,
    "y": 337.889435,
  },
  {
    "h": 17.86203,
    "isContinuation": true,
    "w": 32.5,
    "x": 380.056335,
    "y": 503.197479,
  },
]
`;

exports[`searchPageText Page [3 / 4] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;

exports[`searchPageText Page [4 / 4] - should search the page text 1`] = `
[
  {
    "h": undefined,
    "isContinuation": false,
    "w": undefined,
    "x": NaN,
    "y": undefined,
  },
]
`;
